bookkeeper-server/src/test/java/org/apache/bookkeeper/client/BookKeeperTest.java

```diff
@@ -576,15 +576,15 @@ public void testReadWriteWithV2WireProtocol() throws Exception {
             // basic read/write
             {
                 long ledgerId;
-                try (LedgerHandle lh = bkc.createLedger(digestType, "testPasswd".getBytes())) {
+                try (LedgerHandle lh = bkc.createLedger(1, 1, digestType, "testPasswd".getBytes())) {
                     ledgerId = lh.getId();
                     for (int i = 0; i < numEntries; i++) {
                         lh.addEntry(data);
                     }
                 }
                 try (LedgerHandle lh = bkc.openLedger(ledgerId, digestType, "testPasswd".getBytes())) {
                     assertEquals(numEntries - 1, lh.readLastConfirmed());
-                    for (Enumeration<LedgerEntry> readEntries = lh.readEntries(0, numEntries - 1);
+                    for (Enumeration<LedgerEntry> readEntries = lh.readEntries(0, numEntries, -1);
```

Comment:
We should create a new test instead of modifing the existing tests.

Reply:
Ok, I will create more new tests for the batch read.